src/main.js

```diff
@@ -921,26 +925,21 @@ const DiscoveryFeedKnowledgeArtworkCard = new Lang.Class({
     },
 
     _init: function(params) {
-        params.visible = true;
         this.parent(params);
 
-        let card = new DiscoveryFeedActivatableFrame({
-            content: new DiscoveryFeedContentCardLayout({
-                content: new DiscoveryFeedContentPreview({
-                    image_stream: this.model.thumbnail,
-                    min_width: CONTENT_PREVIEW_LARGE,
-                    min_height: CONTENT_PREVIEW_LARGE
-                }),
-                description: new DiscoveryFeedAppContentDescription({
-                    title: this.model.title,
-                    synopsis: 'by ' + this.model.author,
-                    app_name: 'Masterpiece of the Day'.toUpperCase()
-                }),
-                layout_direction: this.model.layout_direction
-            })
-        });
-        this.add(card);
         this.get_style_context().add_class('artwork');
+    },
+
+    createDescription: function() {
+        return new DiscoveryFeedAppContentDescription({
+            title: this.model.title,
+            synopsis: 'by ' + this.model.author,
+            app_name: 'Masterpiece of the Day'.toUpperCase()
+        });
+    },
```

Comment:
Not going to block on this, but my general preference here is to prefer injecting things like this into the base class as opposed to overriding.